cmake/bob_robotics.cmake

```diff
@@ -660,6 +631,43 @@ endfunction()
 
 function(BoB_modules)
     foreach(module IN LISTS ARGV)
+        if(module STREQUAL robots)
```

Comment:
So, just to clarify, this logic only happens if you just specify the ``robots`` module? Otherwise it's up to you to manually include your robot header etc

Reply:
Yes. This is only for the case when you're using ``ROBOT_TYPE``. I should add a comment explaining this...